src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -348,6 +352,29 @@ export const exchangeStitchingEnvironment = ({
 
     // Resolvers for the above
     resolvers: {
+      CollectorResume: {
+        buyerActivity: {
+          fragment: gql`
+            ... on CollectorResume {
+             userId
+            }
+          `,
+          resolve: async (parent, _args, context, info) => {
+            const exchangeArgs = {
+              buyerId: parent.userId,
+            }
+
+            return await info.mergeInfo.delegateToSchema({
```

Comment:
I see we don't treat any delegateToSchema with try/catch, so I don't think we need any here 👍